src/pages/Build/Research.vue

```diff
@@ -34,8 +34,16 @@ export default {
 	},
 	data() {
 		return {
-			years: [2018, 2017, 2016, 2015, 2014, 2013, 2012, 2011, 2010],
+			years: [2019, 2018, 2017, 2016, 2015, 2014, 2013, 2012, 2011, 2010],
 			papers: [
+				{	year: 2019,
```

Comment:
@go-bears please drop this first key/value to it's own line, then your build should pass.

Reply:
thanks for the review!

Reply:
changed!